services/brig/src/Brig/Options.hs

```diff
@@ -590,10 +590,8 @@ data Settings = Settings
     setSftListAllServers :: Maybe ListAllSFTServers,
     setEnableMLS :: Maybe Bool,
     setKeyPackageMaximumLifetime :: Maybe NominalDiffTime,
-    -- | When set, development API versions are advertised to clients as supported.
-    setEnableDevelopmentVersions :: Maybe Bool,
     -- | Disabled versions are not advertised and are completely disabled.
-    setDisabledAPIVersions :: Maybe (Set Version),
+    setDisabledAPIVersions :: !(Maybe (Set Version)),
```

Comment:
Are you the right person to ask when I want to learn about bang patterns, when to use them, and when not? 🤔